eng/pipelines/templates/steps/template-test-run-job.yml

```diff
@@ -12,8 +12,6 @@ steps:
       parameters:
           Version: $(AzdVersion)
 
-    - template: /eng/pipelines/templates/steps/azd-login.yml
```

Comment:
I have removed this as well in my branch where I am getting this job working on TME with OIDC.